<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiAddFiles</class>
 <widget class="QWidget" name="MultiAddFiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>批量添加文件</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_path">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>查找文件夹</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_selectPath">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
color:blue;</string>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_regular">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_format">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_includeSubPath">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>105</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>遍历子文件夹</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_adaptRegular">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>135</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>文件相对路径匹配正则</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_adaptFormat">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>文件匹配扩展名</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_add">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
